traffic_ops/traffic_ops_golang/region/name.go

```diff
@@ -35,7 +35,13 @@ func GetName(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	defer inf.Close()
-	api.RespWriter(w, r, inf.Tx.Tx)(getName(inf.Tx.Tx, inf.Params["name"]))
+
+	regionNames, err := getName(inf.Tx.Tx, inf.Params["name"])
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("getting region: "+err.Error()))
+		return
+	}
+	api.WriteAlertsObj(w, r, http.StatusOK, tc.Alerts{Alerts: []tc.Alert{api.DeprecationWarning("GET /regions?name={{name}}")}}, regionNames)
```

Comment:
this doesnt have deprecation notices on the errors

Reply:
Add deprecation notices to the errors in 2ada94e8f2.